### ISSUE TITLE
[A11yInspect] Fail: 2.2.2 A in /home/runner/work/test-app/test-app/src/App.tsx

### ISSUE THREAD

**Issue by sitaram-r** [sitaram-r/test-app#27]
## ❌ Accessibility Fail

**Rule:** 2.2.2 A
**File:** `/home/runner/work/test-app/test-app/src/App.tsx`
**Line:** 92, **Column:** 1

### Description
Deprecated marquee element used (Critical)

### Context
```html
<marquee>Moving text</marquee>
```

### Selector
`html > body > div > audio > marquee`

---
*Generated by [A11yInspect](https://github.com/barrierbreak/vscode-scally)*
